src/components/TeamCards/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+import db from "./../../firebaseConfig";
+import MemberCard from "./../MemberCard";
+import { useTranslation } from "react-i18next";
+import { Row } from "antd";
+import "./index.css";
+
+const TeamCards = () => {
+  const [t] = useTranslation();
+  const [dataState, setDataState] = useState([]);
+
+  const fetchData = async () => {
+    const res = await db.collection("team").doc("teamInfo").get();
```

Comment:
i think you can just inline this function in line 20, write a comment on line 18 that says // Fetch team info data from firebase. It's stored in firebase because <include a reason here>

Reply:
I tried it and it gave me the following warning,

`Line 12:13:  Effect callbacks are synchronous to prevent race conditions. Put the async function inside:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state
`

Reply:
I kept it as it is for now but can change it if there is a solution to remove the warning. I will look it up